pkgs/top-level/all-packages.nix

```diff
@@ -16078,9 +16078,12 @@ with pkgs;
   openjdk = jdk;
   openjdk_headless = jdk_headless;
 
+  graalvmPackages =
+    recurseIntoAttrs (callPackage ../development/compilers/graalvm { });
   graalvmCEPackages =
     recurseIntoAttrs (callPackage ../development/compilers/graalvm/community-edition { });
   graalvm-ce = graalvmCEPackages.graalvm-ce;
+  graalvm-oracle = callPackage ../development/compilers/graalvm/oracle { };
```

Comment:
This used to be called GraalVM EE, now just "Oracle GraalVM". Maybe `oracle-graalvm` would be a better name here?